goldendict.py

```diff
@@ -289,11 +369,12 @@ def goldenCardsInfo(self, query, desiredFields):
 
     ac.AnkiConnect.copyToClipboard = copyToClipboard
     ac.AnkiConnect.sendToAnki = sendToAnki
-    ac.AnkiConnect.goldenCardsInfo = goldenCardsInfo
-    
+    if doCrossProfileSearch:
+        ac.AnkiConnect.goldenCardsInfo = crossProfileBank
+    else:
+        ac.AnkiConnect.goldenCardsInfo = goldenCardsInfo
```

Comment:
I don't think we should overwrite the existing method if we don't have to - it may be confusing. Can we instead just call the appropriate function dynamically from inside goldenCardsInfo? Open to alternatives.

Reply:
Oh yeah, that might be cleaner

Should I do something like 

editing `def goldenCardsInfo()`
```
        result = sorted(result, key=lambda k: (k['reps'], k['due']), reverse=True)
        suspended = sorted(suspended, key=lambda k: (k['queue'], k['due']), reverse=False)
        new = sorted(new, key=lambda k: (k['queue'], k['due']), reverse=False)

        if doCrossProfileSearch:
                crossProfileResults = _crossProfileBank(self, query, desiredFields)
        else:
                crossProfileResults = []

        return result + new + suspended + crossProfileResults 
```

I'll change the name of my function to `_crossProfileBank` just to make it clear it isn't intended to be used outside the module / and is designed to function as a static function? (not sure I'm using the right terminology here sry bout that, python noob here). I think I might be able to make `_crossProfileBank` as a static function though since it doesn't use ankiconnect, I'll test it tomorrow

Reply:
Beautiful. All of that makes sense to me & looks reasonable.